webpack.common.js

```diff
@@ -175,5 +179,9 @@ module.exports = {
   },
   output: {
     path: path.resolve(__dirname, 'dist'),
+    clean: {
+      keep: /images/,
+    },
```

Comment:
Do we need this? While currently it works with Gesso out of the box, we also allow adding font files to the gitignore (which would be wiped out by this) and I sometimes have /js/library files I add to the gitignore. We can add a comment to the gitignore to be sure to add anything you want to keep to this file as well, but want to make sure we need this before suggesting we add that.

Reply:
Hmm, good question. Normally I would say yes, we need it, to ensure that webpack doesn't keep generating new versions of the files and keeping the old ones forever. But since we're not including a cache-busting hash in any of our JS or CSS files, we might be okay without it. I'll turn it off for now, and we can add it back if we start seeing old files lingering unnecessarily.

Reply:
Updated!